editor/src/clj/editor/gui.clj

```diff
@@ -2157,15 +2156,6 @@
   [_ evaluation-context node-id old-name new-name]
   (update-basic-gui-resource-reference evaluation-context node-id :particlefx old-name new-name))
 
-;; This InternalTextureNode is a drop-in replacement for TextureNode below.
-;; It can be used in place of TextureNode when you already have a gpu-texture.
-(g/defnode InternalTextureNode
-  (property name g/Str)
-  (property gpu-texture TextureLifecycle)
-  (output texture-infos GuiResourceTextureInfos (g/fnk [name] (sorted-map name {})))
-  (output texture-page-counts GuiResourcePageCounts (g/fnk [name] {name nil})) ; Use a nil texture-page-count to disable validation against the material page count.
```

Comment:
Also, there was a `texture-page-counts` output, but it was unused, since it was never connected to anything. `attach-texture` below didn't connect this output for the internal texture node.